data/json/items/gun/robofac_gun.json

```diff
@@ -290,7 +290,7 @@
     "integral_weight": "650 g",
     "integral_longest_side": "12 cm",
     "longest_side": "42 cm",
-    "shot_spread_multiplier_modifier": 1.25,
+    "shot_spread_multiplier_modifier": 0.25,
```

Comment:
```suggestion
```
As we discussed, it's not an actual thing, so no increased spread for shorter barrel